### PR TITLE
Add helmet SSR

### PR DESCRIPTION
**Pull request by luisherranz**

### CODE REVIEW

**Comment by luisherranz** [comment]
Ready for review 👍

File: packages/core/src/server/templates/html.ts [hunk 1]
```diff
@@ -1,17 +1,21 @@
 import { Template } from "../../types";
 
-const template: Template = ({ html, scriptTags, linkTags }) => `<!doctype html>
+const template: Template = ({ html, frontity, head }) => `<!doctype html>
     <html>
-      <head>
+      <head ${head.htmlAttributes}>
```
Comment (orballo) [inline packages/core/src/server/templates/html.ts]: This line seems wrong. Shouldn't those attributes be rendered inside `<html>`?

Comment (luisherranz) [reply packages/core/src/server/templates/html.ts]: You're right.

**Review by DAreRodz — APPROVED** [review]

File: packages/core/src/server/templates/html.ts (hunk 1 above)
Comment (luisherranz) [reply packages/core/src/server/templates/html.ts]: Fixed

Comment (DAreRodz) [reply packages/core/src/server/templates/html.ts]: Uh, I missed that before approving the PR. Nice catch, @orballo.

**Review by orballo — APPROVED** [review]

**Pull request merged**